public/app/src/canvas.js

```diff
@@ -25,8 +26,13 @@ Canvas.prototype.addCoordinates = function (x, y) {
   this.yAxis.push(y);
 };
 
+Canvas.prototype.getStroke = function () {
+  return this._stroke;
+};
+
 Canvas.prototype.createDot = function (e) {
   this.ctx.beginPath();
   this.ctx.arc(e.clientX, e.clientY, 10, 0*Math.PI, Math.PI*2);
+  this.ctx.fillStyle = this.getStroke().getColour();
```

Comment:
When working with a line, not dot, we'll need to set the style using html canvas's strokeStyle method here instead.